wall_e/src/requirements.txt

```diff
@@ -1,6 +1,6 @@
 yarl<1.2
 bs4
-discord.py[voice]==1.3.4
+discord.py[voice]==1.5.1
```

Comment:
can you please tell *all* the features of wall_e if you want to do a version bump?

Reply:
all commands work. issue with `purgeroles`, but the problem is on master as well. nothing do with version bump